Formula/macvim.rb

```diff
@@ -14,20 +14,29 @@ class Macvim < Formula
   end
 
   option "with-override-system-vim", "Override system vim"
+  option "with-gettext", "Build MacVim with +gettext."
 
   deprecated_option "override-system-vim" => "with-override-system-vim"
 
+  conflicts_with "vim",
```

Comment:
@dunn I don't know if anything has changed, but look at the current version `moreutils` in core. It's baaaack.